rtf_vis_tool/src/Components/gtsfm_edge_list.js

```diff
@@ -0,0 +1,58 @@
+//List of all Edges in GTSFM Visualization Graph
+//imported in LandingPageGraph.js
+
+const gtsfm_edge_list = [['Scene Image Directories', 'Scene Looper'], 
```

Comment:
- Add a map from string to grey nodes, add drawing information and anything else
- Add a map from string to blue nodes, add drawing information and anything else
- the edges below are fine

Reply:
done